README.md

```diff
@@ -429,6 +430,8 @@ or an alternative with environment variable
 
 all variants will return your GCS buckets.
 
+`s5cmd` reads `.aws/credentials` to access Google Cloud Storage. Populate the `aws_access_key_id` and `aws_secret_access_key` fields in `.aws/credentials` with an HMAC key created using this [procedure](https://cloud.google.com/storage/docs/authentication/managing-hmackeys#create).
```

Comment:
It might be helpful to point to an example of `.aws/credentials`, perhaps it is https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html ? This may be obvious for AWS users, but as a someone coming from GCP, I have no idea what that file should look like.

Reply:
Also, the link mentioned above specifies key and secret without quotes. For me, until I added quotes, I was getting a parsing error.

Reply:
As AWS SDK has multiple credentials providers, I think we might add a link to AWS Go SDK's [documentation ](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials) to the https://github.com/peak/s5cmd#specifying-credentials section in a separate PR.